tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -1732,7 +1725,6 @@ def test_placeholder():
     test_forward_crop()
     test_forward_pad()
     test_forward_gather()
-    test_forward_gather_v1()
```

Comment:
This test case will not be executed for sure since ci is using newer tf version, but let's keep it for a while since we still see 'Gather' in a lot of graphdefs in applications

Reply:
test_forward_gather contains testing Gather and GatherV2